hoomd/md/pytest/test_rigid.py

```diff
@@ -453,3 +453,171 @@ def test_rigid_dof(
         thermo_central_free.translational_degrees_of_freedom == (n_bodies + n_free) * 3
     )
     assert thermo_constituent.translational_degrees_of_freedom == 0
+
+
+def test_velocity_constituents_constant_torque(
```

Comment:
Could this test be combined with the next one by parametrizing on the COM velocity?

Reply:
Yes, and I went ahead and did that.